source/computer/macos-setup.rst

```diff
@@ -234,15 +234,27 @@ Windows 和 Linux/macOS 系统下，`文本文件的换行符 <https://www.ruany
 
     $ brew install dos2unix unix2dos
 
-`gawk <https://www.gnu.org/software/gawk/>`__ 是用于文本处理的命令行工具。
-macOS 自带的 awk 语法上与 gawk 有所不同。推荐安装并使用 ``gawk``::
-
-    $ brew install gawk
-
 `wget <https://www.gnu.org/software/wget/>`__ 是用于下载文件的命令行工具::
 
     $ brew install wget
 
+macOS 下自带了很多实用工具，如 ``sed``、``grep`` 等（位于 :file:`/usr/bin/`\ 目录下）。
+需要注意，这些实用工具是由 BSD 提供的，而 Linux 系统下的实用工具则是由 GNU 提供的。
```

Comment:
Links for BSD and GNU?

Reply:
Perhaps not necessary, as these links don't provide any useful information about the utilities.